checks/runchecks.go

```diff
@@ -59,8 +59,8 @@ func getDisabledChecks(ctx *context.Context) (map[string]struct{}, error) {
 	return result, nil
 }
 
-// Exec runs the actions specified and returns the results, without exporting any metrics
-func Exec(ctx *context.Context) ([]*pkg.CheckResult, error) {
+// Exec runs the actions specified and returns the results, without saving artifacts
+func Exec(ctx *context.Context, exportMetrics bool) ([]*pkg.CheckResult, error) {
```

Comment:
This function is used by topology run and kubernetes resource check, was the reason to not export metrics only for topology run to not export metrics if the checks had it defined ?

Reply:
I think in both cases if a custom metric is defined it should be exported - but we shouldn't export the standard check metrics